.github/workflows/multi-node.yml

```diff
@@ -0,0 +1,63 @@
+name: Multi node test
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+    tags-ignore: [v.*]
+jobs:
+  run-multi-node-tests:
+    name: Multi Node Test
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix: # *DO NOT EDIT* the following line is generated by running ./scripts/gh-generate-projects-matrix.sh from the root folder
+        project: [akka-cluster, akka-cluster-metrics, akka-cluster-sharding, akka-cluster-sharding-typed, akka-cluster-tools, akka-cluster-typed, akka-distributed-data, akka-remote-tests]
+      fail-fast: true
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Setup SSH to local
+        run: |
+          mkdir -p ~/.ssh
+
+          # setup the authorized key
+          ssh-keygen -t ed25519 -a 100 -f ${HOME}/.ssh/id_rsa -q -N ""
+          sudo ssh-keyscan -H 127.0.0.1 >> ${HOME}/.ssh/known_hosts
+          sudo ssh-keyscan -H localhost >> ${HOME}/.ssh/known_hosts
+          sudo cat ${HOME}/.ssh/id_rsa.pub >> ${HOME}/.ssh/authorized_keys
+
+          # fix the filesystem permissions
+          sudo chmod go-w ~/
+          sudo chmod 700 ~/.ssh
+          sudo chmod 600 ~/.ssh/authorized_keys
+
+          # restart the ssh daemon
+          sudo service sshd restart
+
+          ssh runner@127.0.0.1 uptime
+      - name: Set up JDK 11
+        uses: olafurpg/setup-scala@v10
+        with:
+          java-version: adopt@1.11.0-9
+      - name: Cache Coursier cache
+        uses: coursier/cache-action@v6.2
+      - name: Multi node test
+        env:
+          JOB_ID: ${{ github.run_id }}
+        run: |
+          sbt -jvm-opts .jvmopts-ci \
+            -Dakka.test.timefactor=1 \
+            -Dakka.cluster.assert=on \
+            -Dsbt.override.build.repos=false \
+            -Dakka.test.tags.exclude=gh-exclude \
+            -Dakka.test.multi-node=true \
+            -Dakka.test.multi-node.targetDirName=${PWD}/target/${JOB_ID} \
+            -Dakka.test.multi-node.java=${JAVA_HOME}/bin/java \
+            -Dmultinode.XX:MetaspaceSize=128M \
+            -Dmultinode.XX:+UseCompressedOops \
+            -Dmultinode.Xms256M \
+            -Dmultinode.Xmx512M \
+            -Dmultinode.host=127.0.0.1 \
+            ${{ matrix.project }}/multiNodeTest
```

Comment:
This will probably need to report failures (e-mail/issue)

Reply:
this is not a nightly, I think is ok to not send emails here

Reply:
Even if it's not nightly, it runs async when there's a new commit to `master` in which case it's good to have feedback as soon as possible.

Reply:
understood, done 👍